geode-lucene/src/integrationTest/java/org/apache/geode/cache/lucene/NestedObjectSeralizerIntegrationTest.java

```diff
@@ -70,7 +77,16 @@ private Region createRegionAndIndexOnInvalidFields() {
     return region;
   }
 
-  private void feedSomeNestedObjects(Region region) throws InterruptedException {
+  protected void feedSomeNestedObjects(Region region) throws InterruptedException {
+    Class clazz = useWhichClass();
+    Constructor<Customer> constructor = null;
+    try {
+      constructor = clazz.getConstructor(String.class, Collection.class, Collection.class,
```

Comment:
Rather than using a Class and finding the constructor through reflection, why not just use a Supplier which returns an instance of the class?